packages/manager/modules/user-account/src/css/source.scss

```diff
@@ -0,0 +1,8 @@
+$flag-icon-css-path: '~flag-icon-css/flags';
+
+@import 'flag-icon-css/sass/flag-icon';
+@import 'bootstrap4/scss/_functions.scss';
```

Comment:
you have imported bootstrap in app as well, do you need it in two places?

Reply:
I have removed this in modules, retained in apps